### PR TITLE
added values, tested and can login to squidex

### PR DESCRIPTION
**Pull request by sanderverhoeven-weareyou**
@SebastianStehle  added several values to the values file, tested the application on local kubernetes cluster can login.

### CODE REVIEW

File: helm/squidex/values.yaml [hunk 1]
```diff
@@ -32,29 +32,71 @@ ingress:
   #      - chart-example.local
 
 env:
-  URLS__BASEURL: null
```
Comment (SebastianStehle) [inline helm/squidex/values.yaml:35]: If we have so many settings we should sort them alphabetically.

File: helm/squidex/values.yaml [hunk 2]
```diff
@@ -32,29 +32,71 @@ ingress:
...
   EVENTSTORE__TYPE: MongoDb
-  IDENTITY__ADMINEMAIL: null
-  IDENTITY__ADMINPASSWORD: null
-  IDENTITY__SHOWPII: true
   LETSENCRYPT_HOST: null
   LETSENCRYPT_EMAIL: null
-  LOGGING__HUMAN: "false"
+  LOGGING__LEVEL: INFORMATION #Trace, Debug, Information, Warning, Error, Fatal
+  LOGGING__HUMAN: true #Setting the flag to true, enables well formatteds json logs.
+  LOGGING__APPLICATIONINSIGHTS__ENABLED: false
```
Comment (SebastianStehle) [inline helm/squidex/values.yaml:43]: Either add all telemetry settings/providers or none.

**Review by SebastianStehle — CHANGES_REQUESTED** [review]

File: helm/squidex/values.yaml (hunk 1 above)
Comment (sanderverhoeven-weareyou) [reply helm/squidex/values.yaml:35]: sorted the env settings. Added more values for the helm show values squidex/squidex cmd. You can export these values which is convenient for deploying the app.

File: helm/squidex/values.yaml (hunk 2 above)
Comment (sanderverhoeven-weareyou) [reply helm/squidex/values.yaml]: added logging settings to values file.

**Comment by DmitriyEsin** [comment]
Hi guys!
I can't add and install your helm chart.
If I’m trying to:
helm repo add squidex https://github.com/Squidex/squidex/tree/master/helm
I got the error:
Error: looks like “https://github.com/Squidex/squidex/tree/master/helm” is not a valid chart repository or cannot be reached: error converting YAML to JSON: yaml: line 124: mapping values are not allowed in this context

**Comment by sanderverhoeven-weareyou** [comment]
> Hi guys!
> I can't add and install your helm chart.
> If I’m trying to:
> helm repo add squidex https://github.com/Squidex/squidex/tree/master/helm
> I got the error:
> Error: looks like “[https://github.com/Squidex/squidex/tree/master/helm”](https://github.com/Squidex/squidex/tree/master/helm%E2%80%9D) is not a valid chart repository or cannot be reached: error converting YAML to JSON: yaml: line 124: mapping values are not allowed in this context

Use :
helm repo add squidex https://squidex.github.io/squidex/helm
helm repo update

This pull request will fix the last label and some values are added.

**Comment by DmitriyEsin** [comment]
Guys, another error :
helm install squidex squidex/squidex --set env.URLS__BASEURL=mydomain --set ingress.hostName=mydomain
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Deployment.spec.template.metadata): unknown field "orleans/clusterId" in io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta, ValidationError(Deployment.spec.template.metadata): unknown field "orleans/serviceId" in io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta]

**Comment by DmitriyEsin** [comment]
Guys, can you increase the initialDelaySeconds parameter?
Seems like it wasn't giving the application enough time to build and it restarts.
I guess maybe it'll be useful for you 
[squidex-7f57ddff67-54h27.log](https://github.com/Squidex/squidex/files/7163703/squidex-7f57ddff67-54h27.log)

**Comment by SebastianStehle** [comment]
Do you have a full? It is really weird.

**Comment by DmitriyEsin** [comment]
Let me reproduce it

**Comment by SebastianStehle** [comment]
Perhaps we should move to startup-probe, but I am not sure how good the general support is.

**Comment by SebastianStehle** [comment]
How exactly have you started Squidex? So far it works on my local docker setup. I am also going to test it with a real kubernetes cluster

**Comment by DmitriyEsin** [comment]
I've fully deleted it from my k8s cluster and deployed it again
I use this command to deploy it `helm install squidex squidex/squidex --set env.URLS__BASEURL=mydomain --set ingress.hostName=mydomain --namespace=mynamespace`
When it is not yet deployed successfuly, it starts to check the livenessProbe parameter infinitely and pod restart, and I get CrashLoopBackOff error
Here is the log and screenshot of events 
[squidex-7f57ddff67-b8w5t.log](https://github.com/Squidex/squidex/files/7163901/squidex-7f57ddff67-b8w5t.log)
Screenshot 2021-09-14 at 20 08 43

**Comment by SebastianStehle** [comment]
Can we move this discussion back to the forum?

**Pull request merged**